appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -5070,4 +5070,83 @@ String newerVersionComponentException(String componentType, int srcCompVersion,
       "<br/>These applications will not run on Android versions older than 4.0.")
   @Description("Text for the Package SDK 26 Warning Dialog Box (HTML)")
   String Package26Notice();
+
+  // Used in editor/youngandroid/properties/YoungAndroidChartTypeChoicePropertyEditor.java
+
+  @DefaultMessage("line")
+  @Description("Text for Chart type choice 'line'")
+  String lineChartType();
+
+  @DefaultMessage("scatter")
+  @Description("Text for Chart type choice 'scatter'")
+  String scatterChartType();
+
+  @DefaultMessage("area")
+  @Description("Text for Chart type choice 'area'")
+  String areaChartType();
+
+  @DefaultMessage("bar")
+  @Description("Text for Chart type choice 'bar'")
+  String barChartType();
+
+  @DefaultMessage("pie")
+  @Description("Text for Chart type choice 'pie'")
+  String pieChartType();
+
+  // Used in editor/youngandroid/properties/YoungAndroidChartPointShapeChoicePropertyEditor.java
+
+  @DefaultMessage("circle")
+  @Description("Text for Chart point shape choice 'circle'")
+  String chartCircleShape();
+
+  @DefaultMessage("square")
+  @Description("Text for Chart point shape choice 'square'")
+  String chartSquareShape();
+
+  @DefaultMessage("triangle")
+  @Description("Text for Chart point shape choice 'triangle'")
+  String chartTriangleShape();
+
+  @DefaultMessage("cross")
+  @Description("Text for Chart point shape choice 'cross'")
+  String chartCrossShape();
+
+  @DefaultMessage("x")
+  @Description("Text for Chart point shape choice 'x'")
+  String chartXShape();
+
+  // Used in editor/youngandroid/properties/YoungAndroidChartLineTypeChoicePropertyEditor.java
+
+  @DefaultMessage("linear")
+  @Description("Text for Chart line type choice 'linear'")
+  String lineTypeLinear();
+
+  @DefaultMessage("curved")
+  @Description("Text for Chart line type choice 'curved'")
+  String lineTypeCurved();
+
+  @DefaultMessage("stepped")
+  @Description("Text for Chart line type choice 'stepped'")
+  String lineTypeStepped();
+
+  // Used in editor/simple/components/MockDataFile.java
+
+  @DefaultMessage("Reading data from {0} to update the columns of {1}.")
+  @Description("Message displayed when data is being parsed for a DataFile component.")
+  String dsataFileParsingMessage(String filename, String dataFile);
+
+  @DefaultMessage("Empty file specified!")
+  @Description("Message displayed when the Source File specified in the DataFile is an empty file.")
+  String emptyFileError();
+
+  // Used in editor/youngandroid/properties/YoungAndroidCsvColumnSelectorPropertyEditor.java
+
+  @DefaultMessage("You must select a column!")
+  @Description("Message displayed when OK button is clicked when there is no column selected.")
+  String noColumnSelected();
+
+  // Used in editor/simple/components Chart Model components
+  @DefaultMessage("Invalid Chart Data entry specified.")
+  @Description("Message displayed when the user enters an invalid Chart Data entry that cannot be parsed.")
+  String invalidChartDataEntry();
```

Comment:
@ewpatton Added ODE Message for invalid chart data entry (on NumberFormatException for Mock Model classes)